CHANGELOG.next.md

```diff
@@ -19,6 +19,7 @@ Thanks, you're awesome :-) -->
 * Added `volume.*` as beta field set. #2269
 * Advanced `process.env_vars` to GA. #2315
 * Advanced `process.io` and `process.tty` fields to GA. #2317
+* Added `threat.indicator.id`. #2252
```

Comment:
```suggestion
* Added `threat.indicator.id`. #2324
```

This should be this PR number, not the issue. Sorry I didn't realize it wasn't the first time I looked at it